proto/one/enterprise/data/dataset_definition_v2.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
+
+message EnterpriseDataSourceDefinition {
+    // DataSoruce name
+    string name = 1;
+
+    // Optional: A list of key value configuration options for the datasource data retrieval
+    map<string, google.protobuf.StringValue> options = 2;
+}
+
+message EnterpriseDataTableQueryParameter {
+    // Name of the datatable query parameter
+    string name = 1;
+
+    // Type of the datatable query parameter
+    string type = 2;
+
+    // Value of the datatable query parameter
+    string value = 3;
+}
+
+message EnterpriseDataTableQuery {
+    // Name of DataTable referenced DataSource
+    string dataSourceName = 1;
```

Comment:
The comment is a little confusing here, is this the DataTable name or the DataSource name?